sphinx/cmdline.py

```diff
@@ -83,6 +84,19 @@ def handle_exception(app, args, exception, stderr=sys.stderr):
                   file=stderr)
 
 
+def jobs_argument(value):
+    # type (int, str) -> int
```

Comment:
Is this typing right? Could you test this?

The reference says as following:
>type= can take any callable that takes a single string argument and returns the converted value:
https://docs.python.org/3.6/library/argparse.html#type

Reply:
Yup. This is broken. It's missing a colon (e.g. `type:`)

Reply:
And that's an annotation for a two-element function anyway - the correct form would be

    # type: (Union[int, str]) -> int

Reply:
@Zac-HD who specifies int value?

Reply:
Nobody!  That was the correct *format* but wrong annotation; in this case the annotation should be:

    # type: (str) -> int